packages/clay-management-toolbar/src/index.tsx

```diff
@@ -0,0 +1,79 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import ActiveState from './ActiveState';
+import classNames from 'classnames';
+import DefaultState from './DefaultState';
+import React from 'react';
+import ResultsBar from './ResultsBar';
+
+interface IProps
+	extends React.ComponentProps<typeof DefaultState>,
+		React.ComponentProps<typeof ResultsBar>,
+		React.ComponentProps<typeof ActiveState> {
+	active?: boolean;
+	className?: string;
+	showResultsBar?: boolean;
+}
+
+const ClayManagementToolbar: React.FunctionComponent<IProps> = ({
+	active = false,
+	className,
+	clearLabel = 'Clear',
+	clearResultsURL,
+	filterLabels,
+	onClearButtonClick = () => {},
```

Comment:
no-op needed?

Reply:
I would say the same as above.

Reply:
hey @bryceosterhaus i won't add this on conditional to show up `ResultsBar` because we have `showResultsBar` we will be imposing that just setting `showResultsBar` to true, developers know what they want.